shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -682,6 +691,14 @@ - (void)setIsGpuDisabled:(BOOL)value {
   _isGpuDisabled = value;
 }
 
+- (void)setupQualityOfService:(flutter::TaskRunners&)task_runners {
```

Comment:
This needs to be done for all threads managed by the engine.  This includes the IO thread as well as the worker task runners (`QOS_CLASS_BACKGROUND`). I suggest exposing this via the platform view creation callback. That way, all platforms are given the opportunity to set thread priorities on all threads managed by the engine.